tools/add_contributors.py

```diff
@@ -46,7 +46,7 @@
 # update with your GitHub username and path to a file with GitHub token
 UPDATE_AVATARS = False
 GH_USERNAME = "Remi-Gau"
-TOKEN_FILE = None
+TOKEN_FILE = "/home/remi/Documents/tokens/gh_user.txt"
```

Comment:
no absolute paths should be hardcoded IMHO

Reply:
should there be an env var or alike to define it?

Reply:
probably unless the API calls done in here can be all handled be handled by the github CLI package (not sure)